packages/ui/src/memberships/modals/SwitchMemberModal/SwitchMemberModal.tsx

```diff
@@ -32,11 +33,11 @@ export const SwitchMemberModal = () => {
   }
 
   const filteredMembers = useMemo(() => {
+    const sortedMembers = members.sort(Comparator<Member>(false, 'handle'))
```

Comment:
There is already a function to sort memberships - `sortMemberships`, consider using it.

Build fail fix:
```suggestion
    const sortedMembers = members.sort(Comparator<Member>(false, 'handle').string)
```